beacon-chain/db/filesystem/blob.go

```diff
@@ -148,6 +153,7 @@ func (bs *BlobStorage) Get(root [32]byte, idx uint64) (blocks.VerifiedROBlob, er
 	if err != nil {
 		return blocks.VerifiedROBlob{}, err
 	}
+	defer blobFetchLatency.Observe(float64(time.Since(startTime).Milliseconds()))
```

Comment:
Now that I think about this, i think that the arguments will be immediately evaluated and only the outer most function is deferred. You have to wrap the whole thing in an anon fn.

https://go.dev/play/p/esKy1B0H-Hz

```suggestion
	defer func() { blobFetchLatency.Observe(float64(time.Since(startTime).Milliseconds())) }()
```